renderer/components/editor/options/fps.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+export default () => (
+  <React.Fragment>
```

Comment:
Fragment syntax? `<React.Fragment>` => `<>`

Reply:
![image](https://user-images.githubusercontent.com/5027156/40330057-07590da4-5d4c-11e8-9b11-6a4c3ad83e3d.png)
Doesn't work

Reply:
We're using `"next": "^4.2.1",` any reason  you did not use latest next @karaggeorge ?

Reply:
We should upgrade to latest, I think it's already at v6 now.

Reply:
Yup, but I think there's some incompatibility with electron-next, will try to upgrade after we get this PR merged.